seed/static/seed/js/controllers/column_settings_controller.js

```diff
@@ -378,7 +365,7 @@ angular.module('BE.seed.controller.column_settings', [])
           });
 
           spinner_utility.show();
-          $q.all(promises).then($scope.complete_column_update, function (data) {
```

Comment:
Here's my thinking. The function `$scope.complete_column_update` calls the org service endpoints we are removing. If we were to remove the `organization_service` calls it would just be running `column_update_complete` with a spinner.